packages/ngtools/webpack/src/compiler_host.ts

```diff
@@ -435,6 +435,10 @@ export class WebpackCompilerHost implements ts.CompilerHost {
       return this.readFile(fileName);
     }
   }
+
+  trace(message: string) {
+    console.log(message);
```

Comment:
I don't think we should use `console.log` for this, but not sure what else we should use. The webpack compilation only allows for warnings and errors, and they show only at the end of the compilation.

@clydin WDYT?